README.md

```diff
@@ -664,16 +664,25 @@ Other Style Guides
       a = 1;
     }
 
+    // bad
     function f2(a) {
-      if (!a) { a = 1; }
+      if (!a) { 
+        a = 1; 
+      }
     }
 
-    // good
-    function f3(a) {
```

Comment:
We should add in an example of how to do the bad examples above (the ones the with variable `a`) correctly or just remove the bad examples above and only have examples that use `opts`